perun-core/src/main/resources/schema.sql

```diff
@@ -857,7 +857,7 @@ create table tasks (
 	status varchar(16) not null,
 	start_time timestamp,
 	end_time timestamp,
-	engine_id integer not null,
+	engine_id integer,
```

Comment:
Please update DB version also in ``test-data.sql`` in perun-core resources. Test DB can't update its own version, since version mismatch prevents perun to startup.